web/router.go

```diff
@@ -147,6 +147,10 @@ func v2Routes(app *services.ChainlinkApplication, engine *gin.Engine) {
 
 	authv2 := engine.Group("/v2", authRequired(app.Store))
 	{
+		uc := UserController{app}
+		authv2.PATCH("/user/password", uc.UpdatePassword)
+		// FIXME: "/account/balance" should probably go here at some point
```

Comment:
On it: https://github.com/smartcontractkit/chainlink/commit/ccc535fe2bfe0e0aa37171e0301131fbe3a31b3e